web/pages/api/v0/market/[id]/comment.ts

```diff
@@ -0,0 +1,39 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import {
+  CORS_ORIGIN_MANIFOLD,
+  CORS_ORIGIN_LOCALHOST,
+} from 'common/envs/constants'
+import { applyCorsHeaders } from 'web/lib/api/cors'
+import { fetchBackend, forwardResponse } from 'web/lib/api/proxy'
+import { htmlToRichText } from 'common/util/parse'
+import { micromark } from 'micromark'
+
+export const config = { api: { bodyParser: true } }
+
+export default async function route(req: NextApiRequest, res: NextApiResponse) {
+  await applyCorsHeaders(req, res, {
+    origin: [CORS_ORIGIN_MANIFOLD, CORS_ORIGIN_LOCALHOST],
+    methods: 'POST',
+  })
+
+  const { id } = req.query
```

Comment:
Kind of weird that you put the ID in a URL parameter instead of the path but I don't care at all about this.

Reply:
Or just `/api/v0/comment` and everything else goes into the POST parameters?